crazyflie/config/server.yaml

```diff
@@ -1,5 +1,7 @@
 /crazyflie_server:
   ros__parameters:
+    firmware_params:
```

Comment:
In crazyflies.yaml, we enable/disable logging with `firmware_logging:` and `enable_logging:`. This would actually make me expect this option to be crazyflies.yaml instead. Or would you rather enable/disable logging in server.yaml eventually as well?

Reply:
I find this annoying as well. To put it in `crazyflies.yaml` would require doing a backwards incompatible file format change. Do you think that would be fine?

Reply:
ahhh.... yes i know what you mean... I guess that is an initial design flaw of us directly starting with the full on parameters in 'firmware_params'

It's not ideal, but perhaps we can solve that in the documentation for now. For the future if we release it for real, then we would need to be more strict about it and perhaps provide some guide about the .yaml files updates between versions. But for now we can accept it as it is, and if we both agree that it should be in crazyflie.yaml, then let's do it now instead of later 🙂

Reply:
Agreed - I added #222 and #224 to track this.